pkg/bridge/ai/caller.go

```diff
@@ -562,3 +624,9 @@ func transToolMessage(msgs []openai.ChatCompletionMessage) []ai.ToolMessage {
 	}
 	return toolMessages
 }
+
+func recordTTFT(ctx context.Context, tracer trace.Tracer) {
+	_, span := tracer.Start(ctx, "TTFT")
+	span.End()
+	time.Sleep(time.Millisecond)
```

Comment:
Notice: We have to use `time.Sleep()` here to guarantee `TTFT` is before than `TBT` when render on baselime.